src/language-services/commonTypes.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import type { DocumentUri, TextDocument } from "vscode-languageserver-textdocument";
+import type { Diagnostic, Position, Range } from "vscode-languageserver-types";
+
+export type { Diagnostic, DocumentUri, Position, Range, TextDocument };
+
+export const enum DiagnosticSeverity {
+    Error = 1,
+    Warning = 2,
+    Information = 3,
+    Hint = 4,
+}
```

Comment:
Is there a reason you're recreating DiagnosticSeverity instead of importing/re-exporting DiagnosticSeverity?

Reply:
Good question. I think it was because I had been playing with type only imports, which wouldn't work for `DiagnosticSeverity`. Resolved this with a second import.